tests/std/tests/P0553R4_bit_rotating_and_counting_functions/test.cpp

```diff
@@ -105,6 +105,27 @@ constexpr bool test_rotr() {
     return true;
 }
 
+// tests functions for 64bit operands that
+// have either high or low halves as zero
+// these may be split into two operations
+// on 32bit platforms and we need to check
+// if we handle the == zero or == ones cases
+// correctly
+constexpr bool test_64bit_split_ops() {
+    constexpr unsigned long long zero_one = 0x0000'0000'FFFF'FFFF;
+    constexpr unsigned long long one_zero = 0xFFFF'FFFF'0000'0000;
+    assert(popcount(zero_one) == 32);
+    assert(popcount(one_zero) == 32);
+    assert(countr_zero(zero_one) == 0);
+    assert(countr_zero(one_zero) == 32);
+    assert(countl_zero(zero_one) == 32);
+    assert(countl_zero(one_zero) == 0);
+    assert(countr_one(zero_one) == 32);
+    assert(countr_one(one_zero) == 0);
+    assert(countl_one(zero_one) == 0);
+    assert(countl_one(one_zero) == 32);
+    return true;
+}

```

Comment:
There should be a newline after this function. (We omit empty lines between functions only when they are extremely short and highly repetitive.)